language-support/java/codegen/src/it/java-1.3/com.digitalasset/AllTests.java

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+        OptionalTest.class,
```

Comment:
We don't need `OptionalTest` for 1.3, as nothing has changed in that regard.
`MapTest.class` needs to be added.

Reply:
Personally I would keep the `OptionalTest` to test for regressions (although upstream modules should be also be doing this). The copy-n-paste isn't ideal and the structuring of the DAML-LF version specific tests could (should) be improved.

Reply:
Agreed. But let's fix the test setup in another PR without copy-pasta. wdyt?

Reply:
Also good.